containers/allan_dip_bot/run_bot.py

```diff
@@ -274,6 +282,13 @@ def main() -> None:
     parser.add_argument(
         "--outdir", type=Path, help="output directory for game json to be stored"
     )
+    parser.add_argument(
+        "--aggressiveness",
+        type=str,
+        choices=["A", "M", "F"],
+        default="M",
```

Comment:
Can these options be changed to `aggressive`, `moderate`, and `friendly`? A letter isn't as readable as a full word.

Reply:
done